v1/linkml-schemas/lifeEvents.yaml

```diff
@@ -82,10 +82,10 @@ slots:
     required: true
   deathRegistrationTime:
     description: Date/time the death was registered, may be the same as the `toe` JWT claim
-    range: StructuredDateTimeClass
+    range: datetime
```

Comment:
It looks like we created `StructuredDateTimeClass`, and we're the only ones using it, so we should probably get rid of that as well?

Reply:
Good point - fixed - thanks